rotkehlchen/types.py

```diff
@@ -560,3 +560,8 @@ def deserialize_from_db(cls, entry: Tuple[int, str, str, str, int, int]) -> 'Use
             last_update_timestamp=Timestamp(entry[4]),
             is_pinned=bool(entry[5]),
         )
+
+
+class EvmTokenKind(DBEnumMixIn):
+    ERC20 = 1
```

Comment:
use auto

Reply:
I will change this to auto but since this is a constant that goes to the database I wanted to be explicit here. auto by default starts at 1 so there shouldn't be any problem unless the default changes

Reply:
Ah it's going to the DB as enum. So leaving it as is is also fine then. They should start from 1.